be/CMakeLists.txt

```diff
@@ -751,6 +766,10 @@ add_subdirectory(${SRC_DIR}/udf)
 add_subdirectory(${SRC_DIR}/tools)
 add_subdirectory(${SRC_DIR}/util)
 
+if (${WITH_DLA_CACHE} STREQUAL "ON")
+    add_subdirectory(${SRC_DIR}/cache)
```

Comment:
I suggest to use directory like `dla_cache`. You can imagine if  another team want to build a general purpose cache but have to use another name, it's very frustrating.